test/unit/spec/OktaSignIn_spec.js

```diff
@@ -5,11 +5,11 @@ define([
   'util/Logger'
 ],
 function (Widget, Expect, Logger) {
-  var signIn;
+  var signIn, devLogger;
   var url = 'https://foo.com';
 
   beforeEach(function(){
-    spyOn(Logger, 'warn');
+    devLogger = spyOn(Logger, 'warn');
```

Comment:
Would prefer leaving it as before, so that we are clear about what needs to be called right on the `expect` statement.

Reply:
This is required due to the update in [`Expect.js`](https://github.com/okta/okta-signin-widget/pull/466/files#diff-9f742f61fcd34d4a03a359547d1eebfcR31), allowing us to reassign the `Logger.warn` spy multiple times.